gradle/dokka.gradle.kts

```diff
@@ -9,11 +9,11 @@ import java.net.*
 apply<DokkaPlugin>()
 //apply<JavaPlugin>()
 
-fun GradleDokkaSourceSetBuilder.makeLinkMapping(projectDir: File) {
+fun GradleDokkaSourceSetBuilder.makeLinkMapping(projectDir: File, branch: String = "master") {
     sourceLink {
         val relPath = rootProject.projectDir.toPath().relativize(projectDir.toPath())
         localDirectory.set(projectDir.resolve("src"))
-        remoteUrl.set(URL("https://github.com/kotlin/kotlinx.coroutines/tree/master/$relPath/src"))
+        remoteUrl.set(URL("https://github.com/kotlin/kotlinx.coroutines/tree/$branch/$relPath/src"))
```

Comment:
I'm still not sure why it's required.

The links should always point to `master` -- if Dokka's results are published, then all related changes are in `master`. Every other branch cannot be the source of truth

Reply:
fixed (but i am not sure that tests would pass for now, lets see)